optaplanner-wb-screens/optaplanner-wb-guided-rule-editor/optaplanner-wb-guided-rule-editor-backend/src/main/java/org/optaplanner/workbench/screens/guidedrule/backend/server/plugin/MultiConstraintHardSoftMatchPersistenceExtension.java

```diff
@@ -0,0 +1,163 @@
+package org.optaplanner.workbench.screens.guidedrule.backend.server.plugin;
```

Comment:
@mcimbora Check your IntelliJ settings with the droolsjbpm-bootstrap readme. It should add the license for you automatically (with the correct year, so this year's year) on java files. It's not part of code style in intellij's setting, but a separate config item if I recall correctly.

- All pure code files definitely need a license header (so all java files, all js files, all scala files, etc).
- As for XML and JSON files, it gets a bit blurry: a list_of_persons.xml hold data, but a properties.xml holds configuration properties and a pom.xml holds build code. Do they all need a license header or just some of them? My advice is to follow drools-wb's style for now.
- Same discussion for CSS files - but add the argument that we like to minimize the payload do the client too...